src/installer/corehost/cli/json/rapidjson/cursorstreamwrapper.h

```diff
@@ -46,7 +46,7 @@ class CursorStreamWrapper : public GenericStreamWrapper<InputStream, Encoding> {
     // counting line and column number
     Ch Take() {
         Ch ch = this->is_.Take();
-        if(ch == '\n') {
+        if (ch == '\n') {
```

Comment:
This is a harmless change, but I'd prefer if rapidjson were kept as a pristine copy from upstream, as it's a 3rd-party component.  (If we need to actually fix things in there, I'd prefer that we also sent patches their way instead of maintaining a fork.)

Reply:
OK @lpereira  I'll back off the change from this PR.
I've submitted a PR for rapidjson: https://github.com/Tencent/rapidjson/pull/1685
We should subsequently fix it in our code too. 
I usually do `if(`  `for(` `while(` searches before submitting PRs, and it would be nice to not hit these cases every time.